src/ActionInterface.cpp

```diff
@@ -291,3 +291,46 @@ ActionInterface::SendUIState()
 
   main_window->SetUIState(GetUIState());
 }
+
+void
+ActionInterface::SetActiveFrequency(const RadioFrequency & freq, const TCHAR * freq_name, bool to_devices)
+{
+  /* update interface settings */
+
+  SetComputerSettings().radio.active_frequency = freq;
+  if(freq_name != nullptr) {
```

Comment:
If `freq_name==nullptr`, then the old name will persist - is that what you want?

Reply:
Implemented freq_name.clear() in else block